Formula/emqx.rb

```diff
@@ -0,0 +1,45 @@
+class Emqx < Formula
+  desc "MQTT broker for IoT"
+  homepage "https://www.emqx.io/"
+  url "https://github.com/emqx/emqx/archive/refs/tags/v5.0.11.tar.gz"
+  sha256 "5b797f768961abccf9ea0050613c02b7d1fc1d3877306d592e72c64a8d95588b"
+  license "Apache-2.0"
+  head "https://github.com/emqx/emqx.git", branch: "master"
+
+  depends_on "autoconf"  => :build
+  depends_on "automake"  => :build
+  depends_on "cmake"     => :build
+  depends_on "coreutils" => :build
+  depends_on "erlang@24" => :build
```

Comment:
There are a few things left in emqx as of v5.0.11 to fix before it can support latest Erlang/OTP (a couple of things were made obsolete in that release). Should be fixed in emqx 5.0.12 or 5.0.13.

Reply:
I'll update the formula as soon as emqx can be officially built with OTP 25.

Reply:
Can you add code comments about this? We can also keep a watch over it.